python_modules/dagster/dagster/core/execution/context/compute.py

```diff
@@ -312,6 +312,15 @@ def selected_output_names(self) -> AbstractSet[str]:
                 selected_outputs.add(output_name)
         return selected_outputs
 
+    def output_asset_key(self, output_name: str = "result") -> AssetKey:
```

Comment:
love the functionality, don't like the name, as it seems a bit too "verby" ("am I telling dagster to output an asset key?").

"asset_key_for_output"?